pkgs/development/haskell-modules/with-packages-wrapper.nix

```diff
@@ -95,6 +95,14 @@ symlinkJoin {
         makeWrapper ${ghc}/bin/$prg $out/bin/$prg --add-flags "${packageDBFlag}=${packageCfgDir}"
       fi
     done
+
+    if [[ -x "${ghc}/bin/haddock" ]]; then
```

Comment:
Oh, could you please add a reference to https://github.com/NixOS/nixpkgs/issues/36976 into the source code a comment? That makes it easier for people who are reading the source code to grok why that wrapper is necessary.

Reply:
Oops, github got confused by my (too eager) squash. Here's the comment:

    # haddock was referring to the base ghc, https://github.com/NixOS/nixpkgs/issues/36976